packages/expo-splash-screen/android/src/main/res/values/styles_splashscreen.xml

```diff
@@ -2,9 +2,18 @@
     Settings in this file should/will be overridden by specific app's resources.
 -->
 <resources>
-  <style name="Theme.App.SplashScreen" parent="Theme.AppCompat.Light.NoActionBar">
-    <item name="android:windowBackground">@drawable/splashscreen</item>
-    <item name="android:windowDrawsSystemBarBackgrounds">true</item>
-    <item name="android:statusBarColor">@android:color/transparent</item>
-  </style>
+  <style name="Theme.App.SplashScreen" parent="Theme.SplashScreen">
+   <!-- Set the splash screen background, animated icon, and animation duration. -->
+   <item name="windowSplashScreenBackground">#ffffff</item>
+
+   <!-- Use windowSplashScreenAnimatedIcon to add either a drawable or an
+        animated drawable. One of these is required. -->
+   <item name="windowSplashScreenAnimatedIcon">@drawable/splashscreen_image</item>
```

Comment:
Using `@drawable/splashscreen_image` and `windowSplashScreenBackground` will not work as you expect.
Android enforce strict rules regarding SplashScreen API assets.

Using app icon without an icon background, you will have a 288x288dp circle mask to put your logo. Everything outside this circle wil be cropped ([one of the many related issues](https://github.com/zoontek/react-native-bootsplash/issues/327))

- [More infos](https://developer.android.com/develop/ui/views/launch/splash-screen#dimensions)
- [MaskedDrawable.kt](https://github.com/androidx/androidx/blob/androidx-main/core/core-splashscreen/src/main/java/androidx/core/splashscreen/MaskedDrawable.kt)

Note that the branding image (`android:windowSplashScreenBrandingImage`) is, at max, a 200x80dp rectangle. It's not polyfilled in the AndroidX core splashscreen library, so only available on Android >= 12 (it is in bootsplash own polyfill)

So I think the best option is to update how the `splash` config is shaped in `app.json` to have:

- `backgroundColor` (set `<item name="windowSplashScreenBackground"><-- THIS VALUE --></item>`)
- `logo`: the logo asset path, converted to multiple drawables for `windowSplashScreenAnimatedIcon`
- `logoWidth`: a number to adjust, max is `288` (dp), no issue under `192` as it's in the circle, warning if `> 192 && <= 288`
- `brand`: the brand asset path, converted to multiple drawables for `android:windowSplashScreenBrandingImage`
- `brandWidth`: a number to adjust, max is `200` (dp) - maybe warn if the image, at 200dp width has a height > 80dp?

`brand` and `brandWidth` could be added later, as don't forget, you will have to implement it on iOS too - I suggest you focus on having `backgroundColor`, `logo` and `logoWidth` working for the moment.

`resizeMode` and `image` params could be removed. Or maybe, if you never implement `brand`, you can keep `image` for the logo (with `imageWidth`?)

![splash-screen-icon-dimensions](https://github.com/expo/expo/assets/1902323/4c821ca9-0da2-489e-aa0d-f505b61ac899)